presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -18,6 +18,7 @@
     public static final String BIG_QUERY = "experimental_big_query";
     private static final String OPTIMIZE_HASH_GENERATION = "optimize_hash_generation";
     private static final String DISTRIBUTED_JOIN = "distributed_join";
+    private static final String TASK_WRITER_COUNT = "task_writer_count";
```

Comment:
I think task_writer_count is more accurate, since it is the per task count, not the global writer count.


> On Mar 31, 2015, at 5:02 PM, Nileema <notifications@github.com> wrote:
> 
> In presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java:
> 
> > @@ -18,6 +18,7 @@
> >      public static final String BIG_QUERY = "experimental_big_query";
> >      private static final String OPTIMIZE_HASH_GENERATION = "optimize_hash_generation";
> >      private static final String DISTRIBUTED_JOIN = "distributed_join";
> > +    private static final String TASK_WRITER_COUNT = "task_writer_count";
> call this String WRITER_COUNT = "writer_count'
> 
> —
> Reply to this email directly or view it on GitHub.
>